js/windowMgr.js

```diff
@@ -174,6 +175,28 @@ function doCreateMainWindow(initialUrl, initialBounds) {
 
     mainWindow.on('closed', destroyAllWindows);
 
+    // Manage File Downloads
+    mainWindow.webContents.session.on('will-download', (event, item, webContents) => {
```

Comment:
Good catch. Have handled it now for all popouts (including help, IMs and Rooms). One case I am unsure about is how do we handle downloads for 3rd party app poputs (ChartIQ for ex.). Do we use our manager for that as well? Please share your thoughts on this.

![screen shot 2017-05-31 at 2 47 06 pm](https://cloud.githubusercontent.com/assets/27778544/26624944/2e91ce82-4610-11e7-8d3c-c5c550e9294a.png)